common/spec/dependabot/clients/azure_spec.rb

```diff
@@ -109,4 +125,17 @@
       end
     end
   end
+
+  describe "#get" do
+    context "Using auth headers" do
+        token = ":test_token"
+        encoded_token = Base64.encode64(":test_token").delete("\n")
+        bearer_token = "test_token"
+
+        include_examples 'Successfully completes get request with auth headers', {"token_type" => "basic non encoded", "credentials" => [{"type" => "git_source", "host" => "dev.azure.com", "token" => token}], "headers" => { "Authorization" => "Basic #{encoded_token}"}}
```

Comment:
I think our linter is failing on the line-length here, there's a hard limit at 80 characters. I'd split up the hash over multiple lines and/or assign them to some variables outside of the example (using `let` seems most idiomatic in our codebase).